system/Model.php

```diff
@@ -257,13 +257,13 @@ protected function doFirst()
 
         if ($this->tempUseSoftDeletes) {
             $builder->where($this->table . '.' . $this->deletedField, null);
-        } elseif ($this->useSoftDeletes && empty($builder->QBGroupBy) && $this->primaryKey) {
+        } elseif ($this->useSoftDeletes && ($builder->QBGroupBy === []) && $this->primaryKey) {
```

Comment:
Is `()` around `$builder->QBGroupBy === []`  is forced by our tooling? Seems unnecessary.